wicket-core/pom.xml

```diff
@@ -207,6 +179,24 @@ org.apache.wicket.validation.validator;-noimport:=true
 				<groupId>org.primefaces.extensions</groupId>
 				<artifactId>resources-optimizer-maven-plugin</artifactId>
 			</plugin>
+			<!--
```

Comment:
@gluser1357 I believe commented-out code need to be dropped

Reply:
Yes, thanks. Code is removed.